src/js/mixins/visiblity.js

```diff
@@ -0,0 +1,28 @@
+import document from 'global/document';
+
+const visibilityMixin = function(object, options) {
```

Comment:
It's possible that this should have a better name, but I just wanted to get the idea out here. Perhaps `backgroundTabMixin` and `hide`/`show` as options.